docs/src/main/sphinx/installation/cli.rst

```diff
@@ -314,10 +314,12 @@ To access a Trino cluster configured to use :doc:`/security/jwt`, use the
 Kerberos authentication
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-In addition to the options that are required when connecting to an unauthorized
-Trino coordinator, invoking the CLI with Kerberos support enabled requires a
-number of additional command line options. The simplest way to invoke the CLI is
-with a wrapper script.
+The Trino :doc:`/installation/cli` can connect to a Trino cluster that has
```

Comment:
Aren't we IN that file now? So this is a circular cross-ref. It can just be text: `Trino CLI`.